.github/workflows/build_package.yml

```diff
@@ -122,23 +122,23 @@ jobs:
             build-family: linux-x86_64
             build-package: main-dist-linux
             experimental: false
-          - runs-on: ah-ubuntu_22_04-c7g_4x-50
+          - runs-on: ubuntu-24.04-arm
```

Comment:
cc @banach-space the arm-hosted runners are queued on https://github.com/iree-org/iree/actions/runs/15724393328 so we're going to try switching to github-hosted runners, at least for this release workflow. That's matching how other release workflows use github-hosted runners.

The https://github.com/iree-org/iree/blob/main/.github/workflows/ci_linux_arm64_clang.yml workflow still uses arm-hosted runners (more powerful machines that we could use for more frequent CI jobs)

Reply:
Yep!